gen/patient.go

```diff
@@ -38,15 +38,10 @@ func Patient(r *rand.Rand, idx int) Object {
 	return patient
 }
 
+var genders = []string{"male", "female"}
```

Comment:
We should also test for "other" and "unknown" values since the biobanks may use these. https://www.hl7.org/fhir/valueset-administrative-gender.html

Reply:
Yes I thought about this but decided to not include that in this PR. I've opened #10 for this.